DEVELOPMENT.md

```diff
@@ -30,28 +36,69 @@ This requires that `kind` is installed and that either `podman` or `docker` is a
-> Commands to do so is `./scripts/local-cluster.sh cache <create|delete>`, then one can make use of the local cluster profiles `<single|multi>-node-cache` that are prepared to use it by default.
+> The command to do so is `./scripts/local-cluster.sh cache <create|delete>`, then one can make use of the local cluster profiles `<single|multi>-node-cache` that are prepared to use it by default.
+
+As a first step, configure the clusters:
 
 ```sh
 # with CK8S_CONFIG_PATH and CK8S_PGP_FP set
 ./scripts/local-cluster.sh config <name> <apps-flavor> <domain>
-./scripts/local-cluster.sh create <name> <kind-config|local-cluster-profile> # use list profiles to see built-in configs
 ```
 
-This will configure apps with `ck8sCloudProvider: none` and `ck8sFlavor: <apps-flavor>` and set some default values to run on local clusters.
-By default it will use `calico` for networking, `local-path-provisioner` for block storage, and `minio` for object storage.
+This will use the default local listen address of `127.0.64.43` for the SC worker node and `127.0.64.143` for the WC worker node.
+
+Apps will be configured with `ck8sCloudProvider: none` and `ck8sFlavor: <apps-flavor>` and set some default values to run on local clusters.
+By default, local clusters will use `calico` for networking, `local-path-provisioner` for block storage, and `minio` for object storage.
+
+We'll also be passing the `--skip-minio` options for the WC, to deploy block storage only to the SC cluster.
+
+```sh
+./scripts/local-cluster.sh create <name>-sc <kind-config-file|local-cluster-profile>
+./scripts/local-cluster.sh create <name>-wc <kind-config-file|local-cluster-profile> --skip-minio
+```
+
+> [!important]
+> To support the integrated SC <--> WC scenario we require that the cluster names be suffixed with either `-sc` or `-wc`
+
+#### In-cluster DNS setup
+
+Making the clusters "aware" of each other will require custom DNS configuration snippets to both the SC and WC that ensure:
+
+- `(grafana|harbor|opensearch|ops).<domain>` resolve to an SC worker node IP from within the WC cluster
+- all other `.<domain>` queries from within the WC cluster resolve to the service IP of the `nginx-ingress`
+- `(grafana|harbor|opensearch|ops).<domain>` resolve to the service IP of the `nginx-ingress` within the SC cluster
+- all other `.<domain>` queries from within the SC cluster resolve to a WC worker node IP
+
+Since these snippets depend on the IP addresses of running containers, they must be setup _after_ the initial `create` commands.
+
+The following command will configure node-local DNS and (re)deploy the `node-local-dns` stack:
 
-The configuration contains some `set-me`'s that must be configured manually.
+```sh
+./scripts/local-cluster-sh setup_node_local_dns
+```
+
+#### Self-signed certificate setup
+
+Add the following block to `$CK8S_CONFIG_PATH/common-config.yaml` (you might need to merge the keys in `global` map manually):
```

Comment:
Suggestion, very much optional:

You can add some additional thing here, that "if you cannot setup cert-manager to use DNS-01 challenges you have the option of using self-signed certificates" or something.